app/src/main/java/app/nameplaceholder/sensorsapp/App.kt

```diff
@@ -10,6 +12,7 @@ class App : Application() {
         super.onCreate()
         instance = this
         bindNotificationState(this)
+        ContextCompat.startForegroundService(this, Intent(this, ForegroundService::class.java))
```

Comment:
This changes behavior of whole application. I had no intention to do tracking when Application (but not Activity) is started

Reply:
Ahh, ok. I just thought it was boring having to start the sensors at every run while I was testing the app. :-)